test/pom.xml

```diff
@@ -239,6 +263,23 @@ THE SOFTWARE.
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin> <!-- TODO pending JENKINS-45271 fix, would be best to finish moving MavenModuleSet-specific tests to maven-plugin and delete the test dep here -->
```

Comment:
Yeah, needs fix asap. OTOH, how many MavenPlugin-dependent tests do we have in the core? Maybe we should just move them elsewhere and remove the dependency.